tools/virtualization/about.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Virtualization  
+subsection: virtualization  
+section: tools  
+description: Run virtual machines at near-native speeds and manage them at ease  
+---
+
+# Virtualization
```

Comment:
Could we maybe just few general sentences and then add just links to respective sub-pages?

Like here: https://developer.fedoraproject.org/tools/vagrant/about.html

Reply:
Addressed here https://github.com/developer-portal/content/pull/394/commits/c8c7dbab93f7ac23cdbcb93b99cbebdb2da63799